shopify_app.gemspec

```diff
@@ -8,8 +8,9 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6')
+  s.required_ruby_version = ">= 2.2.4"
```

Comment:
Also this is on 7.0.11 which prevents existing users using <= 2.2.4 from upgrading and taking advantage of the scripttag updates... 😿